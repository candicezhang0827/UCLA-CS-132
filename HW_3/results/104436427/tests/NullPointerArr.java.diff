1,3c1,2
< --- Error (ProgramFault) ---
< use of uinitialized value
<   Main() line 3
---
> --- Error (Explicit) ---
> null pointer
