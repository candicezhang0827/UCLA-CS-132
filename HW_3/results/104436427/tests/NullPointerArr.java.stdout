func Main()
	t.0 = x
	if t.0 goto :null1
		Error("null pointer")
	null1:
	t.1 = [t.0]
	PrintIntS(t.1)
	ret
